frontend/src/lib/integrations/slackIntegrationLogic.ts

```diff
@@ -45,4 +45,7 @@ export const slackIntegrationLogic = kea<slackIntegrationLogicType>([
             },
         ],
     }),
+    afterMount(({ actions }) => {
+        actions.loadSlackChannels()
```

Comment:
This will load whenever this logic is used which is not great... Especially given that this logic is used elsewhere. I think this should be manually called by the component that needs it when it mounts (and also only if there is a valid slack integration)

Reply:
![2024-12-05 at 10 53 34](https://github.com/user-attachments/assets/8d3627ef-4d20-4398-8e8d-02580016a114)

Will load slack channels through a useEffect from the Helper instead.